string Name
int32 Roll
string email
uint32 Position_in_Ranking